packages/fast-components-react-msft/src/button/button.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import { Foundation, HandledProps } from "@microsoft/fast-components-react-base";
+import { IMSFTButtonHandledProps, IMSFTButtonManagedClasses, IMSFTButtonUnhandledProps } from "./button.props";
+import { IManagedClasses, IMSFTButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    default as BaseButton,
+    IButtonClassNameContract,
+    IButtonHandledProps,
+    IButtonManagedClasses,
+    IButtonUnhandledProps,
+} from "@microsoft/fast-components-react-base/src/button";
+
+/* tslint:disable-next-line */
+class Button extends Foundation<IMSFTButtonHandledProps & IManagedClasses<IMSFTButtonClassNameContract>,  React.AllHTMLAttributes<HTMLElement>, {}> {
```

Comment:
All of the interfaces refer to MSFTButton - why not just refer to this as MSFT button? Should we worry about confusion?

Reply:
We had a lot of talks surrounding this. It is a button with MSFT styles. The entire purpose was to add styles/props on top of, but not create another named button. BUT, that is what the conversation will be about

Reply:
Should the class not match the props?

Reply:
At this point the only thing different is the class - so I'm curious if we agree that this distinction is still merited, or if we should just call it MSFTButton

Reply:
We are going to have a meeting offline about this today before Nick leaves.